### ISSUE TITLE
Swaync toggle/center covers the whole screen.[Bug]

### ISSUE THREAD

**Issue by cknyakina** [ErikReider/SwayNotificationCenter#240]
My notification center was working well, but I don't know what I messed up, but now it covers the whole screen. How can I get it to be shown on the far right end side like it was before? Right now, the dark blue part will cover the desktop wallpaper or whatever app that I am using.
![screenshot_2023-04-05_00-38-09_576624892](https://user-images.githubusercontent.com/81164719/229981535-4690a4cc-61fa-45d7-ad14-514cf28ea5b6.png)

**Comment by ErikReider** [comment]
Could you provide your config?

**Comment by cknyakina** [comment]
> Could you provide your config?

Sure thing: I don't know how to link files so I'll copy the whole thinge here.

> config.json

{
  "$schema": "/etc/xdg/swaync/configSchema.json",
  "positionX": "right",
  "positionY": "top",
  "control-center-margin-top": 10,
  "control-center-margin-bottom": 0,
  "control-center-margin-right": 10,
  "control-center-margin-left": 0,
  "notification-icon-size": 64,
  "notification-body-image-height": 100,
  "notification-body-image-width": 200,
  "timeout": 10,
  "timeout-low": 5,
  "timeout-critical": 0,
  "fit-to-screen": false,
  "control-center-width": 500,
  "control-center-height": 1028,
  "notification-window-width": 500,
  "keyboard-shortcuts": true,
  "image-visibility": "when-available",
  "transition-time": 200,
  "hide-on-clear": false,
  "hide-on-action": true,
  "script-fail-notify": true,
  "scripts": {
    "example-script": {
      "exec": "echo 'Do something...'",
      "urgency": "Normal"
    }
  },
  "notification-visibility": {
    "example-name": {
      "state": "enabled",
      "urgency": "Low",
      "app-name": "Spotify"
    }
  },
  "widgets": [
    "menubar",
    "buttons-grid",
    "volume",
    "backlight",
    "mpris",
    "title",
    "dnd",
    "notifications"
  ],
  "widget-config": {
    "title": {
      "text": "Notifications",
      "clear-all-button": true,
      "button-text": "Clear All"
    },
    "dnd": {
      "text": "Do Not Disturb"
    },
    "label": {
      "max-lines": 1,
      "text": "Controll Center"
    },
    "mpris": {
      "image-size": 100,
      "image-radius": 12
    },
    "volume": {
      "label": ""
    },
    "backlight": {
      "label": ""
    },
    "menubar": {
      "menu#power": {
        "label": "  Power Menu",
        "position": "left",
        "actions": [
          {
            "label": " Lock",
            "command": "swaylock"
          },
          {
            "label": "󰗽 Logout",
            "command": "swaymsg exit"
          },
          {
            "label": " Suspend",
            "command": "systemctl suspend"
          },
          {
            "label": " Reboot",
            "command": "systemctl reboot"
          },
          {
            "label": " Shutdown",
            "command": "systemctl poweroff"
          }
        ]
      }
    },
    "buttons-grid": {
      "actions": [
        {
          "label": "",
          "command": "pactl set-sink-mute @DEFAULT_SINK@ toggle"
        },
        {
          "label": "",
          "command": "pactl set-source-mute @DEFAULT_SOURCE@ toggle"
        },
        {
          "label": "",
          "command": "iwgtk"
        }
      ]
    }
  }
}



> style.css

@define-color foreground #F8F8F2;
@define-color background #120024;
@define-color background-alpha #120024;
@define-color accent #753AB0;
@define-color current-line #250C3E;
@define-color comment #6272A4;

.notification-row {
  transition: all 200ms ease;
  outline: none;
  border-radius: 80px;
}

.control-center .notification-row:focus,
.control-center .notification-row:hover {
  opacity: 1;
  background: transparent;
}

.notification-row:focus .notification,
.notification-row:hover .notification {
  box-shadow: 0 1px 3px 1px rgba(0, 0, 0, 0.5);
  border: 0px solid @accent;
  background: @current-line;
}

.control-center .notification {
  box-shadow: none;
}

.control-center .notification-row {
  opacity: 0.5;
  margin: -5px;
}

.notification {
  transition: all 200ms ease;
  border-radius: 5px;
  margin: 10px;
  box-shadow: 0 1px 3px 1px rgba(0, 0, 0, 0.5);
  padding: 0;
}

.notification button {
  min-height: 20px;
  border-left: 1 solid @accent;
  border-right: 2 solid @accent;
  border-radius: 5px;
}

.notification button:hover {
  background: @accent;
}

.notification-content {
  background: transparent;
  padding: 15px;
  border-radius: 5px;
}

.notification-default-action,
.notification-action {
  padding: 4px;
  margin: 0;
  box-shadow: none;
  background: @current-line;
  border: 2px solid @accent;
  color: @foreground;
}

.notification-default-action:hover,
.notification-action:hover {
  -gtk-icon-effect: none;
  background: @current-line;
}

.notification-default-action {
  border-radius: 5px;
}

.notification-action {
  border-radius: 5px;
  border-top: none;
  border-right: none;
}

.image {
}

.body-image {
  margin-top: 6px;
  background-color: @foreground;
  border-radius: 5px;
}

.summary {
  font-size: 16px;
  font-weight: bold;
  background: transparent;
  color: @foreground;
  text-shadow: none;
}

.time {
  font-size: 16px;
  font-weight: bold;
  background: transparent;
  color: @foreground;
  text-shadow: none;
  margin-right: 18px;
}

.body {
  font-size: 15px;
  font-weight: normal;
  background: transparent;
  color: @foreground;
  text-shadow: none;
}

.top-action-title {
  color: @foreground;
  text-shadow: none;
}

.control-center-clear-all {
  color: @foreground;
  text-shadow: none;
  background: @background;
  border: 2px solid @accent;
  box-shadow: none;
  border-radius: 5px;
}

.control-center-dnd {
  border-radius: 5px;
  background: @background;
  border: 1px solid @accent;
  box-shadow: none;
}

.control-center-dnd:checked {
  background: @accent;
}

.control-center-dnd slider {
  background: @background;
  border-radius: 5px;
}

.control-center {
  background: @background-alpha;
  border-radius: 5px;
  background-clip: border-box;
  padding: 4px;
  box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.3), 0 1px 3px 1px rgba(0, 0, 0, 0.7),
    0 2px 6px 2px rgba(0, 0, 0, 0.3);
  color: @foreground;
  border: 2px solid @accent;
}

.blank-window {
  background: transparent;
}

.widget-menubar,
.widget-buttons-grid {
  margin: 5px;
  padding: 5px;
  border-radius: 5px;
  background: @current-line;
}

.widget-menubar > box > .menu-button-bar > button {
  margin: 5px;
  padding: 5px 160px;
  border: 1 solid @accent;
  font-size: 20px;
  background: #3d2358;
  border-radius: 10px;
}

.widget-menubar button,
.widget-buttons-grid > flowbox > flowboxchild > button {
  margin: 5px;
  padding: 5px 50px;
  border: 1 solid @accent;
  font-size: 20px;
  background: #3d2358;
  border-radius: 10px;
}

.widget-title {
  margin: 8px;
  font-size: 1.5rem;
}

.widget-title > button {
  font-size: initial;
  color: @foreground;
  text-shadow: none;
  background: @background;
  border: 2px solid @accent;
  box-shadow: none;
  border-radius: 5px;
}

.widget-title > button:hover {
  background: @background;
}

.widget-dnd {
  margin: 8px;
  font-size: 1.1rem;
}

.widget-dnd > switch {
  font-size: initial;
  border-radius: 5px;
  background: @background;
  border: 1px solid @accent;
  box-shadow: none;
}

.widget-dnd > switch:checked {
  background: @accent;
}

.widget-dnd > switch slider {
  background: @background;
  border-radius: 5px;
  border: 1px solid @foreground;
}

.widget-label {
  margin: 8px;
  font-size: 20px;
}

.widget-label > label {
  font-size: 20px;
}

.widget-mpris {
  margin: 0px;
}

.widget-mpris button {
  border-radius: 5px;
  background: @current-line;
  margin: 5px;
}

.widget-mpris-player {
  padding: 8px;
  margin: 5px;
  background-color: @current-line;
  border-radius: 5px;
  color: @foreground;
}

.widget-mpris-title {
  font-weight: bold;
  font-size: 1.25rem;
}

.widget-mpris-subtitle {
  font-size: 1.1rem;
}

.widget-volume {
  padding: 5px 20px;
  margin: 5px;
  background-color: @current-line;
  border-radius: 5px;
  color: @foreground;
  font-size: 25px;
}

.widget-backlight {
  padding: 5px 20px;
  margin: 5px;
  background-color: @current-line;
  border-radius: 5px;
  color: @foreground;
  font-size: 25px;
}

**Comment by diggit** [comment]
I tried your config and style and I can't reproduce behavior you are seeing on swaync 0.8.0. (Archlinux, sway and wlroots from git)

**Comment by cknyakina** [comment]
Yes, It was covering the whole monitor. The issue is resolved. After undoing what I had done prior to this, I finally found the issue that caused this. 

This came about when I tried to theme Nautilus file manager, as none of my GTK themes would work on it. I got it to work, following suggestions online that I should edit the gtk4 configs to match that of the theme (found in the _.themes_). That was what was causing the issue for me. When I reverted to what was, everything works fine now. Thank you for all the help. Swaync is awesome and deeply integrated in my workflow, and I am glad it works as it should now.
